### PR TITLE
[master < T1080] overload << operator for mgp::Type

### PR DESCRIPTION
**Pull request by mpintaric55334**
During writing collections query module, the need to be able to print the mgp::Type was found, so it was solved with this PR.

[master < Task] PR
- [ ] Check, and update documentation if necessary
- [x] Provide the full content or a guide for the final git message

To keep docs changelog up to date, one more thing to do:
- [X] Write a release note here 
You are now able to print the type of mgp::Type() using operator << . For example, if you have a mgp::List() list, cout<< list <<endl will output "list".

- [X] Tag someone from docs team in the comments

### CODE REVIEW

File: include/mgp.hpp [hunk 1]
```diff
@@ -3185,6 +3226,44 @@ inline bool Value::IsDuration() const { return mgp::value_is_duration(ptr_); }
...
+    case Type::Date:
+      return "Type: Date";
+    case Type::LocalTime:
+      return "Type: LocalTime";
+    case Type::LocalDateTime:
+      return "Type: LocalDateTime";
+    case Type::Duration:
+      return "Type: Duration";
+    default:
+      return "Type: Unknown";
```
Comment (antoniofilipovic) [inline include/mgp.hpp:3263]: can you throw an error here?

File: include/mgp.hpp [hunk 2]
```diff
@@ -3185,6 +3226,44 @@ inline bool Value::IsDuration() const { return mgp::value_is_duration(ptr_); }
 inline bool Value::operator==(const Value &other) const { return util::ValuesEqual(ptr_, other.ptr_); }
 
 inline bool Value::operator!=(const Value &other) const { return !(*this == other); }
+
+inline const std::string Value::TypeString() const {
+  const mgp::Type &valueType = Type();
+
+  switch (valueType) {
+    case Type::Null:
+      return "Type: Null";
```
Comment (antoniofilipovic) [inline include/mgp.hpp:3235]: Maybe it makes sense to just return Null, Node, etc.. so user can decide on what kind of error message will be written after that

Comment (antoniofilipovic) [reply include/mgp.hpp:3235]: So instead of Type::Null, return just Null, etc. Does that make more sense?

**Review by antoniofilipovic — CHANGES_REQUESTED** [review]
I would change that we return "<type>" instead of "type: <type>", otherwise looks good

**Comment by antoniofilipovic** [comment]
@mpintaric55334 there is one other thing I haven't thought about until now. We can support instead of TypeString, operator<< which serves for output: https://learn.microsoft.com/en-us/cpp/standard-library/overloading-the-output-operator-for-your-own-classes?view=msvc-170. This is one example, what do you think?

File: include/mgp.hpp (hunk 2 above)
Comment (mpintaric55334) [reply include/mgp.hpp:3235]: Yeah, will do

**Comment by mpintaric55334** [comment]
@vpavicic will update the docs when I get the green light from Antonio.

**Review by antoniofilipovic — APPROVED** [review]
Looks good to go

**Comment by vpavicic** [comment]
Is this also C++ ?

**Comment by mpintaric55334** [comment]
Yes, it is

**Pull request merged**